src/app/blog/iroh-dag-sync/page.mdx

```diff
@@ -0,0 +1,293 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import Image from 'next/image'
+
+export const post = {
+  draft: false,
+  author: 'Rüdiger Klaehn',
+  date: '2024-07-01',
+  title: 'Iroh and Ipld',
+  description:
+    'A possible extension for iroh to deal with DAGs and IPLD data',
+}
+
+export const metadata = {
+  title: post.title,
+  description: post.description,
+}
+
+export default (props) => <BlogPostLayout article={post} {...props} />
+
+## Problem
+
+By default, iroh has a very minimalist set of primitives for content-addressed
+data.
+
+We offer BLAKE3 hashed blobs of arbitrary size, with a very rich set of
+capabilities including verified range requests, verified streaming and size
+proofs.
+
+In addition, we offer a single container type, a sequence of BLAKE3 hashes.
+
+Using these two primitives, we provide specialized collections of data. A common
+pattern is to use a hash sequence, with the first child being a blob of
+self-describing metadata.
+
+These primitives are sufficiently versatile to handle many use cases, such
+as file system sync in [sendme](https://iroh.computer/sendme).
+
+Focusing on BLAKE3 and the various capabilities that BLAKE3 makes possible is
+the right choice for iroh. But that does not change the fact that sometimes
+you have different needs.
+
+You might want to use different hash functions, for example because you have
+a need for a hash function like [poseidon](https://eprint.iacr.org/2019/458.pdf)
+that is more suitable to use in zero knowledge proofs. Or you have existing
+data that is hashed with the ubiquitous SHA-2 hash.
+
+Also, you might have data that naturally forms a DAG, such as a git commit
+history, unixfs data, event logs, or even blockchains.
+
+Iroh-bytes does not completely solve these use cases, but it provides a very
+useful building block of a possible solution.
```

Comment:
```suggestion
`iroh-blobs` does not completely solve these use cases, but it provides a very
useful building block for a possible solution.
```

I think you are referencing the crate here, correct?

Reply:
Gah, yes. I keep using the old name.

This is nothing. It took me way to long to figure out why `RUST_LOG=iroh_bytes=debug` did not do anything...